application/app/Http/Controllers/Admin/AdminUsersController.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Models\AdminUser;
+
+class AdminUsersController extends Controller
+{
+    /**
+     * @param Request $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\View\View|void
+     */
+    public function index(Request $request)
+    {
+        //オーナー権限ユーザーのみ利用可
+        $loginUser = Auth::guard('admin')->user();
+        if(!$loginUser->is_owner) return \App::abort(403);
+
+        $para = $request->all();
```

Comment:
妙な省略は読み手によっては可読性を下げるので使わないように
```suggestion
        $parameter = $request->all();
```

Reply:
他でいうと`$bldAdminUsers`とか。
これなんの略ですか？
少なくとも私はわからないです。

Reply:
まず配列でやり取りするのは辞めたほうがいいですね。
配列だとIDE等で補完が一切効かないので、保守性が最悪です。
第３者から見た場合に$paraの中に何が入るのかさえ分かりません。
LaravelにはFormRequestクラスが用意されているのでAdminUserIndexRequestクラスなりを実装し、
下記のような形でQueryを取得出来るようにした方がいいです。
```
class AdminUserIndexRequest extends FormRequest
{
    public function name(): string
    {
        return $this->input('name', '');
    }
}

// example
$request->name();
```